catalyst/dl/experiment/config.py

```diff
@@ -535,6 +539,14 @@ def get_callbacks(self, stage: str) -> "OrderedDict[Callback]":
             if not is_already_present:
                 callbacks[callback_name] = callback_fn()
 
+        # Remove LoggerCallback on workers nodes
+        if get_rank() > 0:
+            to_del = (LoggerCallback, ConfusionMatrixCallback)
+            for k in list(
```

Comment:
Seems that wrapping `filter` statement with `list` is redundant

Reply:
done